.github/workflows/integration.yml

```diff
@@ -1,129 +0,0 @@
-# Run secret-dependent integration tests only after /ok-to-test approval
-on:
-  pull_request:
-  repository_dispatch:
-    types: [ok-to-test-command]
-
-name: Integration tests
-
-jobs:
-  integration-trusted:
```

Comment:
You have removed some checks that were needed, e.g. if the SHA matches, you have also removed the ok-to-test command. These should be reinstated.

Reply:
i personally don't want to be running this /ok-to-test command all the time, and the security risk is irrelevant when we are using a test account where nothing in it matters. We can ban a user who is trying anything malicious anyways. I don't mind adding it back until we can have a proper discussion on it